homework1/task05/tests/test_max_subarr_sum.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+import pytest
+from max_subarr_sum.task05 import find_maximal_subarray_sum
+
+
+@pytest.mark.parametrize(
+    ("a", "k", "expected_result"),
+    [
+        ([1, 3, -1, -3, 5], 2, 4),
+        ([], 5, -sys.maxsize),
+        ([1, 3], 0, 0),
+        ([-1, -3, -1, -3, -5], 2, -4),
+    ],
```

Comment:
Let's test `[3, 3, -6, 1],  3,  6`

Reply:
6 - is incorrect value. 0 is correct.

Reply:
Sorry, you are right! 6 is correct! My code works incorrect. Thanks. I will change code.